featuremanagement/aio/_featurefilters.py

```diff
@@ -3,19 +3,47 @@
 # Licensed under the MIT License. See License.txt in the project root for
 # license information.
 # -------------------------------------------------------------------------
-from abc import abstractmethod
-from .._featurefilters import FeatureFilter as FF
+from abc import ABC, abstractmethod
 
 
-class FeatureFilter(FF):
+class FeatureFilter(ABC):
```

Comment:
Does this need ABC as a param? Isn't it available without placing it here?

Reply:
`ABC` is marking it as an `Abstract Class`